### PR TITLE
Add missing info in datasource-properties.json to Zarr v2 route

### PR DESCRIPTION
**Pull request by MichaelBuessemeyer**
The wklibs use the zarrv2 `datasource-properties.json` route when opening the remote dataset. When this route is called the backend transforms layers to zarr layer. During the conversion process, some properties of layers were omitted. I updated the code to include the parameters that were not passed previously. 
For reference: The conversion code is called here in line 104:
https://github.com/scalableminds/webknossos/blob/97ef1d16c8ae2bdfad62f28808f5f237145aacb3/webknossos-datastore/app/com/scalableminds/webknossos/datastore/controllers/ZarrStreamingController.scala#L94-L109

### URL of deployed dev instance (used for testing):
- https://addndinfotozarrstreamingdatasource.webknossos.xyz/

### Steps to test:
- Get a token to view the following 4d dataset: https://addndinfotozarrstreamingdatasource.webknossos.xyz/datasets/sample_organization/13457227-4d-mag4.local.zarr/view#256,256,17,0,0.826,pos-t=8
  - eg via: view the dataset
  - in the navbar menu select share and copy the link
  - open the link in a new private browser tab (where you are not logged in)
  - Copy the token from the URL params `token=<token>`  
- Check the following datasource-properties.json [https://addndinfotozarrstreamingdatasource.webknossos.xyz/data/zarr/sample_organization/13457227-4d-mag4.local.zarr/datasource-properties.json?token=<my-token>](https://addndinfotozarrstreamingdatasource.webknossos.xyz/data/zarr/sample_organization/13457227-4d-mag4.local.zarr/datasource-properties.json?token=<my-token>). It should include additional axes.

### TODOs:
- [x] Setup dev instance with an nd dataset with default view configs. Both should show in the datasource-properties.json returned by the route :)
- [x] Fix loading nd data chunks

### Issues:
- Needed for but does not solve [#7279](https://github.com/scalableminds/webknossos/issues/7279)

------
(Please delete unneeded items, merge only when none are left open)
- [x] Updated [changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)

### CODE REVIEW

**Review by normanrz — APPROVED** [review]

**Review by normanrz — APPROVED** [review]
Good stuff!

**Comment by MichaelBuessemeyer** [comment]
I'd like to defer this until the zarr3 streaming has been resolved

**Comment by MichaelBuessemeyer** [comment]
@markbader you can check the nd wklibs support with the dev instance deployed for this pr :)

**Comment by MichaelBuessemeyer** [comment]
But be aware that this branch currently has the broken zarr streaming in it. Viewing data is not possible and I do not want to update the branch as this pr should wait anyway until #7995 is merged

**Comment by markbader** [comment]
> you can check the nd wklibs support with the dev instance deployed for this pr :)

Thanks for pushing this forward, wk-libs bounding box looks good now:
`NDBoundingBox(topleft=(0, 0, 0, 0), size=(17, 512, 512, 35), axes=('t', 'x', 'y', 'z'))`

**Comment by MichaelBuessemeyer** [comment]
>Thanks for pushing this forward, wk-libs bounding box looks good now:
NDBoundingBox(topleft=(0, 0, 0, 0), size=(17, 512, 512, 35), axes=('t', 'x', 'y', 'z'))

**Comment by fm3** [comment]
Post-processing the json doesn’t really feel right. Maybe we can find out why these values weren’t there in the first place and maybe fix that? :thinking:

**Comment by MichaelBuessemeyer** [comment]
>Post-processing the json doesn’t really feel right. Maybe we can find out why these values weren’t there in the first place and maybe fix that? 🤔

This is one of the point I'd like to talk with you about :)

**Comment by MichaelBuessemeyer** [comment]
Just tested zarr nd streaming and it works already. I simply requested out of bounds coordinates :see_no_evil: 

To test use: https://addndinfotozarrstreamingdatasource.webknossos.xyz/data/zarr/sample_organization/13457227-4d-mag4.local.zarr/13457227.zarr/1/0.8.8.8.1?token=<my-token>

**Comment by MichaelBuessemeyer** [comment]
@fm3 Could you please have a final look :)? :pray:

File: webknossos-datastore/app/com/scalableminds/webknossos/datastore/dataformats/zarr/ZarrCoordinatesParser.scala [hunk 1]
```diff
@@ -26,12 +28,18 @@ object ZarrCoordinatesParser {
       (x, y, z) = (parsedCoordinates(parsedCoordinates.length - 3),
                    parsedCoordinates(parsedCoordinates.length - 2),
                    parsedCoordinates(parsedCoordinates.length - 1))
-      additionalCoordinates = if (parsedCoordinates.length > 4)
+      reorderedAdditionalAxes = reorderedAdditionalAxesOpt.getOrElse(List.empty)
+      _ <- bool2Fox(reorderedAdditionalAxes.length == parsedCoordinates.length - 4) ?~> "zarr.invalidAdditionalCoordinates" ~> NOT_FOUND
+      sentAdditionalCoordinates = parsedCoordinates.length > 4
```
Comment (fm3) [inline webknossos-datastore/app/com/scalableminds/webknossos/datastore/dataformats/zarr/ZarrCoordinatesParser.scala:33]:
```suggestion
      requestContainsAdditionalCoordinates = parsedCoordinates.length > 4
```
`sent` is not immediately recognizable as a boolean. I always like bools to be obvious in the variable naming :)

**Review by fm3 — APPROVED** [review]
Nice :sunglasses:  I just suggested one more variable renaming.

I did not re-test everything now, but my guess is you tested both zarr2 and zarr3 streaming again? If not, please do go for another round :innocent:

**Comment by MichaelBuessemeyer** [comment]
should work now. [a1b7c43](https://github.com/scalableminds/webknossos/pull/7976/commits/a1b7c435a0156ebaf165e6742113c34b55b13cb4) includes a small fix for not sending the correct zarr version when requesting a `datasource-properties.json` for an annotation with an volume annotation layer.

**Pull request merged**